components/automate-cli/pkg/diagnostics/context.go

```diff
@@ -203,6 +209,73 @@ func (c *testContext) DoLBRequest(path string, opts ...lbrequest.Opts) (*http.Re
 	return nil, lastErr
 }
 
+func (c *testContext) GetVersion() (string, error) {
+	if c.version != "" {
+		return c.version, nil
+	}
+
+	httpResp, err := c.DoLBRequest("/api/v0/version")
+	if err != nil {
+		return "", err
+	}
+	defer httpResp.Body.Close() // nolint: errchech
+
+	resp := struct {
+		Build string `json:"build_timestamp"`
+	}{}
+
+	if err := json.NewDecoder(httpResp.Body).Decode(&resp); err != nil {
+		return "", err
+	}
+
+	c.version = resp.Build
+
+	return resp.Build, nil
+}
+
+func (c *testContext) IsIAMV2() (bool, error) {
```

Comment:
the tricky part here is we can't assume that we'll always be able to check the policy version, since that endpoint is going away with the force-upgrade. 

so when `/policy_version` returns a 404, we can assume either of these cases are true:
1. the Automate instance is pre-IAMv2 beta (so it must be on v1)
2. the Automate instance is post-force-upgrade (so it must be on v2)

Reply:
Sounds like we are planning on leaving the version endpoint for now?